packit-ci.fmf

```diff
@@ -42,20 +42,22 @@
        - /functional/db-postgresql-sanity-on-localhost
        - /functional/db-mariadb-sanity-on-localhost
        - /functional/db-mysql-sanity-on-localhost
+       - /functional/durable-attestion-sanity-on-localhost
        - /functional/ek-cert-use-ek_check_script
        - /functional/ek-cert-use-ek_handle-custom-ca_certs
        - /functional/install-rpm-with-ima-signature
        - /functional/keylime-non-default-ports
        - /functional/keylime_tenant-commands-on-localhost
        - /functional/keylime_tenant-ima-signature-sanity
+       - /functional/measured-boot-policy-sanity
```

Comment:
This one should not be here. I mean, it won't be executed anyway since it requires real TPM device on a system.